test/pkg_autocorrects_test.cc

```diff
@@ -19,95 +19,117 @@ using namespace std;
 auto logger = spdlog::stderr_color_mt("pkg-autocorrects-test");
 auto errorQueue = make_shared<sorbet::core::ErrorQueue>(*logger, *logger);
 
-string examplePackage = "class Opus::ExamplePackage < PackageSpec\nend\n";
+string examplePackage = "class ExamplePackage < PackageSpec\nend\n";
 string examplePackagePath = "example/__package.rb";
 
 namespace sorbet {
-struct TestPackageFile {
-    // the `ParsedFile` corresponding to the package in which we want
-    // to make the edit
-    ast::ParsedFile targetParsedFile;
-    // the `ParsedFile` corresponding to the package that we want to
-    // add.
-    ast::ParsedFile newParsedFile;
-
-    TestPackageFile(ast::ParsedFile targetParsedFile, ast::ParsedFile newParsedFile)
-        : targetParsedFile(move(targetParsedFile)), newParsedFile(move(newParsedFile)) {}
-
-    static TestPackageFile create(core::GlobalState &gs, string filename, string source) {
-        // add the package file
-        vector<core::FileRef> files;
-        {
-            core::UnfreezeFileTable fileTableAccess(gs);
-            files.emplace_back(gs.enterFile(filename, source));
-            files.emplace_back(gs.enterFile(examplePackagePath, examplePackage));
-        }
+string makePackageRB(string name, string strictDeps, string layer, vector<string> imports = {},
+                     vector<string> exports = {}) {
+    string importList = "";
+    for (auto &import : imports) {
+        importList += fmt::format("  import {}\n", import);
+    }
```

Comment:
This could just be a `map_join` in the main `fmt::format` call below -- mapping each with the above, and joining with a zero-length string?